### DISCUSSION TITLE
Add Support for .obj or Common 3D Object File Formats

### DISCUSSION THREAD

**Discussion by donttellmeimcute** [Genesis-Embodied-AI/genesis-world#410] [Ideas]
It would be great to include support for .obj files or other widely-used 3D object file formats. The .obj format is particularly popular due to its simplicity and compatibility across various 3D software.

Adding support for this format would enhance the usability of the project for tasks involving 3D models. If possible, please consider including other common formats like .fbx or .stl as well, as they are frequently used in the industry.

Looking forward to hearing your thoughts or providing further details!

**Comment by zhouxian** [comment]
Obj and stl are already supported
